lib/block-patterns.php

```diff
@@ -202,36 +201,31 @@ function update_core_patterns() {
 		'quote',
 	);
 
-	$new_core_block_patterns = array(
-		'media-text-nature',
-		'two-images-gallery',
-		'three-columns-media-text',
-		'quote',
-		'large-header-left',
-		'large-header-text-button',
-		'media-text-art',
-		'text-two-columns-title',
-		'three-columns-text',
-		'text-two-columns-title-offset',
-		'heading',
-		'three-images-gallery',
-		'text-two-columns',
-		'media-text-arquitecture',
-		'two-buttons',
-	);
-
 	foreach ( $core_block_patterns as $core_block_pattern ) {
-		$name = 'core/' . $core_block_pattern;
-		if ( WP_Block_Patterns_Registry::get_instance()->is_registered( $name ) ) {
-			unregister_block_pattern( $name );
+		unregister_block_pattern( 'core/' . $core_block_pattern );
+	}
+}
+
+/**
+ * Import patterns from wordpress.org/patterns.
+ */
+function load_remote_patterns() {
+	$patterns = get_transient( 'gutenberg_remote_block_patterns' );
+	if ( ! $patterns ) {
+		$request = new WP_REST_Request( 'GET', '/__experimental/pattern-directory/patterns' );
+		$core_keyword_id = 11; // 11 is the ID for "core".
```

Comment:
Don't let this block this PR, but: how can we make this value less magic? Could this option exist more explicitly in the parent API (`/__experimental/pattern-directory/patterns`) rather than just guessing a term identifier?

Reply:
There's a discussion about that at https://github.com/WordPress/gutenberg/pull/28800#discussion_r617986203